TINY-10000: It is now possible to create a list in an editable part within a non-editable root (#8879)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- The `mceInsertClipboardContent` command will now also accept a `files` property to insert images. #TINY-9776

### Fixed

> - Creating lists is possible when the selection is in an editable area inside a non-editable area. #TINY-10000

Reviewer: Always past tense for 'fixed' items. And as past tense I think it reads better swapped around.

```suggestion
- When an editable area was nested inside a non-editable area, creating lists was not possible. #TINY-10000
```
